serverless-configs/aws-functions.yml

```diff
@@ -182,6 +182,16 @@
             type: request
             resultTtlInSeconds: 0
 
+  scaleImage:
+    handler: serverless/src/scaleImage/handler.default
+    timeout: ${env:LAMBDA_TIMEOUT, '30'}
+    events:
+      - http:
+          method: get
+          cors: ${file(./serverless-configs/${self:provider.name}-cors-configuration.yml)}
+          path: scale/{concept_type}/{concept_id}
+          # Not sure if we'll need the entire path come gametime
```

Comment:
Answer the question and remove the comment.

Reply:
I've seen no evidence working this locally or comparing it to the other lambda setups we have to the contrary so I removed this comment